common/beerocks/tlvf/src/src/beerocks_message.cpp

```diff
@@ -61,7 +61,7 @@ message_com::parse_intel_vs_message(ieee1905_1::CmduMessageRx &cmdu_rx)
     if (!cmdu_rx.getMessageLength()) {
         LOG(ERROR) << "cmdu is not initialized!";
     }
-    auto tlv = cmdu_rx.addClass<ieee1905_1::tlvVendorSpecific>();
+    auto tlv = cmdu_rx.getClass<ieee1905_1::tlvVendorSpecific>();
```

Comment:
You're right, this is the first time we change addClass to getClass so that the initial message passing of vendor speficic CMDUs will work. I will add this in a separate commit which will come right after this one.
Regarding line 87 - will fix this also.